rnote-cli/src/cli.rs

```diff
@@ -70,18 +74,51 @@ pub(crate) async fn run() -> anyhow::Result<()> {
             import_file(&mut engine, input_file, rnote_file).await?;
         }
         Commands::Export {
-            rnote_file,
+            rnote_files,
             output_file,
+            output_format,
             with_background,
             with_pattern,
         } => {
             println!("Exporting..");
 
-            // apply given arguments to export prefs
-            engine.export_prefs.doc_export_prefs =
-                create_doc_export_prefs_from_args(&output_file, with_background, with_pattern)?;
+            let format = match (&output_file, output_format) {
+                (Some(file), None) => {
+                    get_export_format(file.extension().and_then(|ext| ext.to_str()))
+                }
+                (_, out_format) => get_export_format(out_format.as_deref()),
+            };
+
+            match format {
+                Ok(format) => {
+                    // apply given arguments to export prefs
+                    engine.export_prefs.doc_export_prefs =
+                        create_doc_export_prefs_from_args(format, with_background, with_pattern)?;
+                }
+                Err(err) => return Err(err),
+            }
 
-            export_to_file(&mut engine, rnote_file, output_file).await?;
+            match output_file {
+                Some(ref output) => match rnote_files.get(0) {
+                    Some(file) => {
+                        if rnote_files.len() > 1 {
+                            return Err(anyhow::anyhow!("Was expecting only 1 file. Use --output-format when exporting multiple files."));
+                        }
+                        export_to_file(&mut engine, file, output).await?
+                    }
+                    None => {
+                        return Err(anyhow::anyhow!("Failed to get filename from rnote_files."))
+                    }
+                },
+                None => {
+                    for file in rnote_files {
+                        let mut output = file.clone();
+                        output.set_extension("pdf");
```

Comment:
I think this shouldn't be hardcoded as pdf, right?

Reply:
oops yes